synctl/cli.py

```diff
@@ -4683,6 +4779,8 @@ def main():
                 print('testid is required')
         elif get_args.op_type == POP_SIZE:
```

Comment:
same with cost, use abbreviations here
```suggestion
        elif get_args.op_type == POP_SIZE or get_args.op_type == 'size':
```